src/main/java/app/gpx_animator/Configuration.java

```diff
@@ -33,7 +33,6 @@
 public final class Configuration {
 
     private static final Color DEFAULT_BACKGROUND_COLOR = Color.white;
-    public static final long DEFAULT_PHOTO_ANIMATION_DURATION = 700L;
```

Comment:
If you really remove this constant, this class should have compile errors. This constant is used in two or three places in this class.